pyproject.toml

```diff
@@ -30,7 +30,7 @@ packages = [
 
 [tool.poetry.dependencies]
 python = "^3.7"
-deepdiff = ">=5.5.0,>=6.2.0"
```

Comment:
There was a reason this was put in, see @chadell for details. It should be something like `">=5.5.0,<6.99.0"`. Ideally it would account for not using 6.0.x and 6.1.x as well, but not sure if it matters or could be done. 

The lock file should also be update for this (and only this) dependency.

Reply:
I changed it to allow exactly this behaviour.